test/views/email/CaptureEmailViewSpec.scala

```diff
@@ -38,7 +38,7 @@ class CaptureEmailViewSpec extends ViewBaseSpec {
     val removeEmail = "summary"
     val removeEmailDesc = ".panel-border-narrow"
     val removeEmailLink = ".panel-border-narrow a"
-    val onlyAddEmail = "#content > article > p"
+    val onlyAddEmail = "#content > article > form > div:nth-child(2) > div > p"
```

Comment:
The corresponding test seems to fail for me. This selector worked for me though: ".form-field > p:nth-child(2)"